test/functional/api/vim_spec.lua

```diff
@@ -5361,4 +5361,27 @@ describe('API', function()
     api.nvim__redraw({ win = 0, range = { 0, -1 } })
     n.assert_alive()
   end)
+
+  it('nvim__complete_set', function()
```

Comment:
this doesn't seem like the right place for this test.

how about `test/functional/ui/popupmenu_spec.lua` ?

Reply:
make sense